EIPS/eip-4672.md

```diff
@@ -0,0 +1,48 @@
+---
+eip: 4672
+title: address.nonce
+description: A new EVM opcode to get the current nonce of an EOA.
+author: 0xfoobar (@0xfoobar)
+discussions-to: https://ethereum-magicians.org/t/eip-4672-account-nonce-opcode/8171
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-02-01
+requires: none
```

Comment:
I believe you can omit `requires` entirely if you don't have any requirements.

Reply:
done